stages/test/test_kickstart.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+
+import os.path
+import shutil
+import unittest
+from tempfile import mkdtemp
+
+from org_osbuild_kickstart import main as kickstart_main
+
+test_cases = [
+    {
+        "options": {"lang": "en_US.UTF-8"},
+        "expected": "lang en_US.UTF-8",
+    },
+    {
+        "options": {"keyboard": "us"},
+        "expected": "keyboard us",
+    },
+    {
+        "options": {"timezone": "UTC"},
+        "expected": "timezone UTC",
+    },
+    {
+        "options": {
+            "lang": "en_US.UTF-8",
+            "keyboard": "us",
+            "timezone": "UTC",
+        },
+        "expected": "lang en_US.UTF-8\nkeyboard us\ntimezone UTC",
+    },
+]
+
+
+class TestOrgOsbuilderKickstart(unittest.TestCase):
+
+    def setUp(self):
+        self.tmpdir = mkdtemp()
```

Comment:
This can be done with `with tempfile.TemporaryDirectory` which I kinda like.

Reply:
I went with the pytest tmp_path fixture here now